plugins/inputs/mongodb/mongodb.go

```diff
@@ -53,6 +54,9 @@ var sampleConfig = `
   ## When true, collect per collection stats
   # gather_col_stats = false
 
+  ## When true, collect stats per collection
```

Comment:
Can you please be a bit clearer on what exactly is collected? The text almost matches the one of `gather_col_stats`...

Reply:
done